charts/ratify/README.md

```diff
@@ -107,7 +107,7 @@ $ helm upgrade -n gatekeeper-system [RELEASE_NAME] ratify/ratify
 | azureWorkloadIdentity.clientId                     | ClientID of AAD application/Managed identity associated with Workload Identity                                                                                                                                                      | ``                             |
 | azureManagedIdentity.clientId                      | ClientID of Managed identity                                                                                                                                                                                                        | ``                             |
 | azureManagedIdentity.tenantId                      | TenantID of Managed Identity resource                                                                                                                                                                                               | ``                             |
-| akvCertConfig.enabled                              | Enables/disables Azure Key Vault certificate store                                                                                                                                                                                  | `false`                        |
+| akvCertConfig.enabled                              | Enables/disables Azure Key Vault certificate store. If you are using custom chart, certificate store should be referenced through a Verifier CR, references in ConfigMap will not be correctly resolved.                                                                                                                                                                   | `false`                        |
```

Comment:
| akvCertConfig.enabled                              | Enables/disables Azure Key Vault certificate store. If you are using a custom chart, certificate store should be referenced through a Verifier CR. References in ConfigMap will not be correctly resolved.                                                                                                                                                                   | `false`                        |